src/storage/src/table/mod.rs

```diff
@@ -151,30 +150,13 @@ impl<T: AsRef<[u8]>> Deref for KeyedRow<T> {
     }
 }
 
-#[try_stream(ok = (Op, OwnedRow), error = StorageError)]
-pub async fn deserialize_log_stream<'a>(
+pub type ChangeLogRow = ChangeLogValue<OwnedRow>;
```

Comment:
Just found it's basically the same as `Record` in common. 🤣

https://github.com/risingwavelabs/risingwave/blob/0be5d74f26526bcaa0b6d18a0cedd627d23cedf7/src/common/src/array/stream_record.rs#L30-L36